Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:0.10
+FROM node:0.12
```

Comment:
I've tested this personally with 0.12. The docker image does support 4, however, I have not personally tested it yet.